tools/bamparse/test-data/table_antisense.tabular

```diff
@@ -0,0 +1,239 @@
+gene	alignment1.bam	alignment2.bam
```

Comment:
This test seems weak ! (only 0, thus low complexity of the data, increased chance to pass the test for bad reasons)